android/src/main/java/com/reactnativestripesdk/StripeSdkModule.kt

```diff
@@ -404,21 +404,50 @@ class StripeSdkModule(reactContext: ReactApplicationContext) : ReactContextBaseJ
 
   @ReactMethod
   fun createToken(params: ReadableMap, promise: Promise) {
-    val type = getValOr(params, "type", null)?.let {
-      if (it != "Card") {
+    val type = getValOr(params, "type", null)
+    type?.let {
```

Comment:
You can use:
`when(type){
   "BankAccount" -> {...}
    null -> {...}
}
`

Reply:
Good idea. I just refactored this function into a big `when` expression